doc/classes/AnimationTreePlayer.xml

```diff
@@ -55,7 +57,7 @@
 			<argument index="1" name="animation" type="Animation">
 			</argument>
 			<description>
-				Set the animation for an animation node.
+				Binds a new animation from the [member master_player] [code]AnimationTreePlayer[/code]'s animation node with name [code]id[/code].
```

Comment:
Not sure if some words are not missing here?
(...) from the [member master_player] **to the** [code]AnimationTreePlayer[/code]'s animation node(...)

Reply:
ah yes, nice catch.